overview/glossary.md

```diff
@@ -11,35 +11,82 @@ sidebarType: 0
 - TOC
 {:toc}
 
-## Account ID
+## Ad Formats
 
-`Prebid Server` - Prebid Server host companies may give out account IDs to publishers for authorization, configuration,
-and reporting purposes. This account ID is provided on s2sConfig when utilizing Prebid Server from Prebid.js and in the init function when utilizing Prebid SDK.
+This section contains terms around ad formats and their respective configuration.
 
-## AdSlot
+### MediaType or Media Format
 
-The 'hole' in the web page or mobile app where an ad may appear. This is very similar to `AdUnit`, but is
-the technical implementation defining which ad sizes and formats may appear within it.
+`banner` - also called 'display', these ads are generally simple rectangular images.
 
-In the "Google Publisher Tag" (GPT), the "defineSlot()" function call maps an HTML div (the technical
-implementation) to the AdUnit, (the sellable item).
+`native` - the components of this mediatype like text, links, and images are all separated so they
+can be formatted to smoothly fit into the content.
 
-## AdUnit
+`video` - see Player-Initiated Video and Page-Initiated Video
 
-`Ad servers such as Google Ad Manager` - the AdUnit is the thing that business people care about. It's an
-inventory concept targeted by [line items](#line-items).
+### Multiformat AdUnit
 
-[Prebid Adunit] - Prebid adds another layer to the ad server AdUnit: which header bidders should participate in the auction and their parameters.
+An AdUnit that can display creatives of multiple media formats: banner, native, and/or video.
+
+### Page-Initated Video
+
+Also called 'outstream', these are ads that can share an AdUnit in Prebid.js with banners, but are displayed with
+a special 'renderer' to play a video ad instead of a static image. Unlike 'player-initiated' units (aka 'instream video'), this renderer only renders the ad and does not show content.
```

Comment:
Having read the GAM documentation https://support.google.com/admanager/answer/9606829?hl=en&ref_topic=9596566&sjid=8628919592789708366-SA, I understand why we group `outstream` into banner, but it's purely because of the way GAM treats outstream today. 
Fundamentally though, outstream is video. Are we building a Product that is meant to work exclusively with GAM ? Or do we want to be Ad Server agnostic ? I would like to decouple prebid as much as possible from GAM, and have translation layers to convert formats to GAM accepted values.

Reply:
Thanks for the review @karimMourra - there've been several discussions on this recently. Prebid defines "outstream" as "in-page video", which is rendered by wrapping it with arbitrary HTML/Javascript rendering logic. This differs from "in-player video", which cannot handle HTML/javascript, and just expects a VAST URL.

In fact, the Prebid Mobile team has agreed to change their implementation of outstream to support renderers. 

This way of looking at the world is not tied to GAM.

Discussed in docs committee - we're going to merge what we have here, which is again, quite general and not tied to GAM. We can definitely pick up this thread again in the video committee to refine the wording.